<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadSchedule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditSchedule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 1, 7);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>EXIT CTC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dispatch">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispatch A Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Map">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Map and Train Info</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutomaticToggle">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="ThroughputValue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="ThroughputUnits">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>People/Hr.</string>
    </property>
   </widget>
   <widget class="QLabel" name="ThroughputText">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Throughput:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCTC_Main_Menu">
    <property name="title">
     <string>CTC Main Menu</string>
    </property>
   </widget>
   <addaction name="menuCTC_Main_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
